docs/tutorials/further_topics/classification_of_strategies.rst

```diff
@@ -24,13 +24,13 @@ This allows us to, for example, quickly identify all the stochastic
 strategies::
 
     >>> len([s for s in axl.strategies if s().classifier['stochastic']])
-    31
+    36
 
 Or indeed find out how many strategy have only use 1 turn worth of memory to
```

Comment:
There's a typo here, should be `how many strategies only use 1 turn worth of memory`, nice if you can fix in this PR but no problem otherwise.